### DISCUSSION TITLE
Mollie Order items

### DISCUSSION THREAD

**Discussion by Web10-Joris** [duncanmcclean/simple-commerce#1031] [Feature Requests]
Nice to have: Extend Mollie to also create/update [Orders](https://docs.mollie.com/orders/overview) inside Mollie. Then generate a Payment based on that. This way you can also see line items inside Mollie and manage the orders there.

**Comment by duncanmcclean** [comment]
This is coming in the [next version of Simple Commerce](https://duncanmcclean.com/future-of-simple-commerce).

**Reply by Web10-Joris** [reply]
Nice! Thanks Duncan 👍 

What about this notice in their docs? I waited with implementing it myself because of this message. Will it be implemented as an extra gateway (`MollieOrdersGateway` and `MolliePaymentsGateway`) or is it replacing the Payments API. 
![CleanShot 2024-10-02 at 18 39 30@2x](https://github.com/user-attachments/assets/9c631e08-5b15-4c0a-9baa-8a1375518deb)

**Reply by duncanmcclean** [reply]
Oh, I haven't seen that notice. Thanks for pointing it out!

I was planning on replacing the existing Mollie gateway, but I guess I'll just leave it as it is.
